packages/fluentui/react-northstar-emotion-renderer/src/disableAnimations.ts

```diff
@@ -20,7 +20,8 @@ export function disableAnimations(styles: ICSSInJSStyle): ICSSInJSStyle {
     if (animationProps.indexOf(property) !== -1) {
       delete styles[property];
     } else if (isStyleObject(styles[property])) {
-      styles[property] = disableAnimations(styles[property] as ICSSInJSStyle);
+      // Cast to any to avoid "error TS2590: Expression produces a union type that is too complex to represent"
+      (styles as any)[property] = disableAnimations(styles[property] as ICSSInJSStyle);
```

Comment:
@layershifter or others, any (heh) ideas for a more appropriate type than `any` here and in the next file? Potentially could be solved by declaring a more specific type for `styles` earlier.

Reply:
I tried and it fails even on a simple assignment:

```
const style: ICSSInJSStyle = {}
styles[property] = style
```

That looks super strange 😳

Reply:
Yikes...do you think we should try to dig in further, or just leave it with the cast since the impact in our repo is limited to two places?